src/System.IO.Abstractions.TestingHelpers/MockDirectoryInfo.cs

```diff
@@ -131,7 +131,15 @@ public override DateTime LastWriteTimeUtc
             set { GetMockFileDataForWrite().LastWriteTime = value.ToLocalTime(); }
         }
 
+#if NET6_0_OR_GREATER
         /// <inheritdoc />
+        public override string LinkTarget
+        {
+            get { return null; }
```

Comment:
I need guidance on this. Where would we set/get this data? I'll take a look tonight to see, but if someone can point me in the right direction is might save some time.

Reply:
I see two options:

1. Keep it as is, returning a fixed `null`, and implement the mock part later
2. Allow defining link targets in [`MockFileData`](https://github.com/TestableIO/System.IO.Abstractions/blob/d7ba4dc9a47ec054730aba16fb1995ad77809cea/src/System.IO.Abstractions.TestingHelpers/MockFileData.cs)/[`MockDirectoryData`](https://github.com/TestableIO/System.IO.Abstractions/blob/d7ba4dc9a47ec054730aba16fb1995ad77809cea/src/System.IO.Abstractions.TestingHelpers/MockDirectoryData.cs)

I'd slightly prefer option `2` because I'm pretty sure people will quickly want to mock the new property, but I'm also totally fine doing that later 😀

Reply:
Updated, but I'll wait until #791 goes in and then I'll update `MockFileInfo` again to match that updated pattern.